third-party/cmake/FetchZetasql.cmake

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 4Paradigm
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set(ZETASQL_HOME https://github.com/jingchen2222/zetasql)
```

Comment:
Can move this repo to [4paradigm](https://github.com/4paradigm/)?

Reply:
@jingchen2222 it should be. For some historic reason we don't transfer the forked repo to 4Paradigm

Reply:
Can update zetasql_home now?

Reply:
yes